format/leveldb/testdata/leveldb_table_uncompressed.fqtest

```diff
@@ -0,0 +1,100 @@
+$ fq -d leveldb_table dv uncompressed.ldb/000005.ldb
+     |00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f|0123456789abcdef|.{}: uncompressed.ldb/000005.ldb (leveldb_table) 0x0-0x65a (1626)
+     |                                               |                |  data[0:1]: 0x0-0x601 (1537)
+     |                                               |                |    [0]{}: data_block 0x0-0x601 (1537)
+     |                                               |                |      uncompressed{}: 0x0-0x5fc (1532)
+     |                                               |                |        entries[0:4]: 0x0-0x5f4 (1524)
+     |                                               |                |          [0]{}: entry 0x0-0x1d4 (468)
+0x000|00                                             |.               |            shared_bytes: 0 0x0-0x1 (1)
+0x000|   13                                          | .              |            unshared_bytes: 19 0x1-0x2 (1)
+0x000|      bd 03                                    |  ..            |            value_length: 445 0x2-0x4 (2)
+     |                                               |                |            key{}: 0x4-0x17 (19)
+0x000|            6c 6f 72 65 6d 2e 64 6f 6c 6f 72   |    lorem.dolor |              user_key: "lorem.dolor" 0x4-0xf (11)
+0x000|                                             01|               .|              type: "value" (0x1) 0xf-0x10 (1)
+0x010|03 00 00 00 00 00 00                           |.......         |              sequence_number: 3 0x10-0x17 (7)
+0x010|                     4c 6f 72 65 6d 20 69 70 73|       Lorem ips|            value: "Lorem ipsum dolor sit amet, consectetur adipisc..." 0x17-0x1d4 (445)
+0x020|75 6d 20 64 6f 6c 6f 72 20 73 69 74 20 61 6d 65|um dolor sit ame|
+*    |until 0x1d3.7 (445)                            |                |
+     |                                               |                |          [1]{}: entry 0x1d4-0x3a2 (462)
+0x1d0|            06                                 |    .           |            shared_bytes: 6 0x1d4-0x1d5 (1)
+0x1d0|               0d                              |     .          |            unshared_bytes: 13 0x1d5-0x1d6 (1)
+0x1d0|                  bd 03                        |      ..        |            value_length: 445 0x1d6-0x1d8 (2)
+     |                                               |                |            key{}: 0x1d8-0x1e5 (13)
+0x1d0|                        69 70 73 75 6d         |        ipsum   |              user_key_suffix: raw bits 0x1d8-0x1dd (5)
```

Comment:
There are not strings?

Reply:
@wader They _are_; I couldn't figure out if you had a scalar method I could plug into `d.FieldRawLen`... since I need the bytes as well:

```
br := d.FieldRawLen("user_key_suffix", int64(unsharedSize-typeAndSequenceNumberSize)*8)
```

Reply:
Aha ok, the suffix bytes are not always valid utf8? anyways i think the current way is fine! but i do wonder if there should be a `Raw` variant that return bytes? (maybe there is?) the reason `Raw` returns a `bitio.BitReader` is that they can be used for very large bit ranges so that the bits will only be read if really needed to save on IO and memory.